ui/src/components/ExportDialog.tsx

```diff
@@ -1,132 +1,246 @@
-import React, {useContext} from "react";
-import {Backdrop, Button, CircularProgress, Grid, TextField, Typography,} from "@mui/material";
+import React, { useContext, useState } from "react";
+import {
+  Backdrop,
+  Button,
+  CircularProgress,
+  FormControl,
+  FormControlLabel,
+  FormLabel,
+  Grid,
+  Radio,
+  RadioGroup,
+  Stack,
+  TextField,
+  Typography,
+} from "@mui/material";
 import Dialog from "@mui/material/Dialog";
 import DialogActions from "@mui/material/DialogActions";
 import DialogContent from "@mui/material/DialogContent";
-import DialogContentText from "@mui/material/DialogContentText";
 import DialogTitle from "@mui/material/DialogTitle";
-import {createDockerDesktopClient} from "@docker/extension-api-client";
+import { createDockerDesktopClient } from "@docker/extension-api-client";
 
-import {MyContext} from "../index";
+import { MyContext } from "../index";
+import { useExportVolume } from "../hooks/useExportVolume";
+import { VolumeOrInput } from "./VolumeOrInput";
+import { ImageAutocomplete } from "./ImageAutocomplete";
+import { useExportToImage } from "../hooks/useExportToImage";
+import { NewImageInput } from "./NewImageInput";
 
-const client = createDockerDesktopClient();
+const ddClient = createDockerDesktopClient();
 
-function useDockerDesktopClient() {
-    return client;
+interface Props {
+  open: boolean;
+  onClose(v: boolean): void;
 }
 
-export default function ExportDialog({...props}) {
-    console.log("ExportDialog component rendered.");
-    const ddClient = useDockerDesktopClient();
-    const context = useContext(MyContext);
+export default function ExportDialog({ open, onClose }: Props) {
```

Comment:
Previously, when saving an image, we had a nice message explaining to users how they could check that the content had been saved successfully in the image.

I think it'd be good to keep it when saving a volume into both a local and a new image, WDYT?

![image](https://user-images.githubusercontent.com/15997951/183048555-ee53a658-44fd-4dfe-bdfd-6d817af2a65b.png)